sdk/src/quic.rs

```diff
@@ -11,7 +11,7 @@ pub const QUIC_MIN_STAKED_CONCURRENT_STREAMS: usize = 128;
 pub const QUIC_TOTAL_STAKED_CONCURRENT_STREAMS: usize = 100_000;
 
 // Set the maximum concurrent stream numbers to avoid excessive streams
-pub const QUIC_MAX_STAKED_CONCURRENT_STREAMS: usize = 2048;
+pub const QUIC_MAX_STAKED_CONCURRENT_STREAMS: usize = 512;
```

Comment:
Maybe just add a comment here justifying this constant value. Other than that, LGTM.

Reply:
Added comments